.github/workflows/semantic-pr.yaml

```diff
@@ -0,0 +1,46 @@
+name: "Lint PR"
+
+on:
+  pull_request_target:
```

Comment:
We **should** not use `pull_request_target`. I've already handled a CVE that lead to creds being leaked because of `pull_request_target` :)

It can leak tokens, secrets (if we add them in the future). There are more details here: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request_target.

If there is no way to assert semantic PR without using `pull_request_target`, I would prefer just having due diligence as part of PR review instead.

Reply:
It's not possible to it without `pull_request_target`

https://github.com/amannn/action-semantic-pull-request?tab=readme-ov-file#event-triggers